src/Okta.Sdk.UnitTests/ApplicationsClientShould.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Linq;
+using FluentAssertions;
+using Okta.Sdk.UnitTests.Internal;
+using Xunit;
+
+namespace Okta.Sdk.UnitTests
+{
+    public class ApplicationsClientShould
+    {
+        [Fact]
+        public void NotReturnNullWhenFeaturesHasNoData()
+        {
+            var mockRequestExecutor = new MockedStringRequestExecutor(GetBookmarkApplicationStubResponse());
+            var client = new TestableOktaClient(mockRequestExecutor);
+
+            var app = client.Applications.GetApplicationAsync<IBookmarkApplication>("foo").Result;
+
+            app.Features.Any().Should().BeFalse();
+        }
+
+        [Fact]
+        public void ReturnEmptyListWhenFeaturesIsNullInTheResponse()
```

Comment:
This test and the one above look the same to me. Should they be testing separate use cases?

Reply:
I forgot the '[]' in the getStub method.